extensions/spark/kyuubi-extension-spark-3-3/src/test/scala/org/apache/spark/sql/ZorderSuiteBase.scala

```diff
@@ -746,14 +746,25 @@ trait ZorderSuiteBase extends KyuubiSparkSQLExtensionTest with ExpressionEvalHel
                 UnresolvedFunction("current_date", Seq.empty, false)),
               UnresolvedRelation(TableIdentifier("p")))))))
 
-    // TODO: add following case support
-    intercept[ParseException] {
-      parser.parsePlan("OPTIMIZE p zorder by (c1)")
```

Comment:
Does Spark SQL support `ORDER BY` with parentheses

Reply:
No, Spark does not support it.

Reply:
then I think we don't need to support that too

Reply:
I think we can delete those code and comments, unless @ulysses-you and @cfmcgrady have other ideas

Reply:
Seems added by @cfmcgrady , I'm fine to just remove the comment

Reply:
Just following the deltalake. https://docs.delta.io/latest/optimizations-oss.html#z-ordering-multi-dimensional-clustering